### PR TITLE
fastboot: add page

### PR DESCRIPTION
**Pull request by f4ww4z**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/fastboot.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# fastboot
+
+> Communicate to connected Android devices when in bootloader mode (the one place adb doesn't work).
```
Comment (sbrl) [inline pages/common/fastboot.md]: Perhaps `Communicate with`.... would read better here?

File: pages/common/fastboot.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Unlock bootloader:
```
Comment (sbrl) [inline pages/common/fastboot.md]: Perhaps `Unlock the bootloader` would read better?

File: pages/common/fastboot.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`fastboot oem unlock`
+
+- Re-lock the bootloader:
```
Comment (sbrl) [inline pages/common/fastboot.md]: Not sure a hyphen is needed here.

File: pages/common/fastboot.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`fastboot oem lock`
+
+- Reboot device from fastboot mode to fastboot mode again:
```
Comment (sbrl) [inline pages/common/fastboot.md]: Perhaps `Reboot the device from fastboot mode into`..... would read better?

File: pages/common/fastboot.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+`fastboot reboot bootloader`
+
+- Flash ASDF.zip:
```
Comment (sbrl) [inline pages/common/fastboot.md]: What precisely is `ASDF.zip` in this case?

File: pages/common/fastboot.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`fastboot flash {{ASDF.zip}}`
+
+- Flash a custom recovery image ASDF.img:
+
+`fastboot flash recovery {{ASDF.img}}`
+
+- Display connected devices:
```
Comment (sbrl) [inline pages/common/fastboot.md]: Perhaps `Display the connected`.... would read better?

File: pages/common/fastboot.md (hunk 1 above)
Comment (agnivade) [inline pages/common/fastboot.md]: Put adb within backticks(``)

File: pages/common/fastboot.md (hunk 5 above)
Comment (agnivade) [reply pages/common/fastboot.md]: It is an image which is to be flashed. I think "Flash a given image" and then change ASDF.zip in the example to file.zip would be better.

File: pages/common/fastboot.md (hunk 6 above)
Comment (f4ww4z) [reply pages/common/fastboot.md]: I think it should be the same as it already makes sense. Doesn't it?

**Comment by f4ww4z** [comment]
@agnivade @sbrl Sorry for the delay. I have made your requested changes. Please take a look :)

File: pages/common/fastboot.md (hunk 6 above)
Comment (sbrl) [reply pages/common/fastboot.md]: It is understandable, yes - but it's more grammatically correct to say `Display the connected devices:`.

**Comment by f4ww4z** [comment]
Just made sure this PR is up to date. @sbrl Do you approve this page?

**Pull request merged**